test/integration/builder/additional.js

```diff
@@ -113,7 +113,7 @@ module.exports = function(knex) {
     });
 
     // TODO: This doesn't work on oracle yet.
-    if (['postgresql', 'mssql'].includes(knex.client.dialect)) {
+    if (['pg', 'mssql'].includes(knex.client.driverName)) {
```

Comment:
Probably we should start using enum for this :)

Reply:
Probably, but I won't be doing that in this PR :) Its a bit out of scope to change all of those to enums. Also now enums won't bring much help anymore, because most of the used string values are validated to be one of available driverNames.